### ISSUE TITLE
Extension Manager: Lack of feedback to the user

### ISSUE THREAD

**Issue by joaoantoniocardoso** [bluerobotics/BlueOS#1398]
Couple of observations along the video:
1. Lack of feedback that the extension was installed/uninstalled.
2. Lack of feedback when extension is starting.
3. ~Edit dialogue state probably lives longer than it should~ (Fixed by #1426)
4. ~Indicates the wrong version on `Update to version X` button.~ (Fixed, tracked [here](https://github.com/bluerobotics/BlueOS-docker/issues/1418))

https://user-images.githubusercontent.com/5920286/214693680-f3dcb7fe-cba9-4d11-8442-a275c618f221.mp4

**Comment by joaoantoniocardoso** [comment]
Reopening as 1 & 2 are still not fixed.

**Comment by rafaellehmkuhl** [comment]
We should up this in the priority.

**Comment by patrickelectric** [comment]
Fixed in #3843, after #1426 and #1432. The operations push notifications and the cards have spinners now.
